src/Endpoint/Partition.php

```diff
@@ -161,7 +161,8 @@ private function resolveRegion($service, $region)
     private function isServicePartitionGlobal($service)
     {
         return isset($this->data['services'][$service]['isRegionalized'])
-            && false === $this->data['services'][$service]['isRegionalized'];
+            && false === $this->data['services'][$service]['isRegionalized']
+            && isset($this->data['services'][$service]['partitionEndpoint']);
```

Comment:
I've seen warnings emitted in the `aws-sdk-php-symfony` build. My understanding of the endpoints.json format when writing the `Partition` class was that a `partitionEndpoint` would always be present if a service self-reported as not regionalized. That doesn't seem to be true in all cases.